package.json

```diff
@@ -46,13 +48,13 @@
     "symlink": "ln -s $PWD/bin/index /usr/local/bin/fly",
     "clean": "rimraf dist",
     "lint": "eslint src test",
-    "coverage": "babel-node node_modules/.bin/isparta cover test/*.js",
+    "coverage": "babel-node --presets es2015,stage-3 ./node_modules/.bin/isparta cover test/*.js",
```

Comment:
Code coverage still doesn't work, but I can't help on that. With the current published version, it doesn't work either, and I don't know how isparta/istanbul work. Sorry!